BEACON-V2-Model/genomicVariations/endpoints.json

```diff
@@ -255,6 +263,80 @@
           "type": "string"
         }
       },
+      "start": {
+        "name": "start",
+        "in": "query",
+        "schema": {
+          "type": "array",
```

Comment:
Wouldn't this be a string when within a URL query param? The string would need to be comprised of comma separated integers